src/main/java/org/betonquest/betonquest/BetonQuest.java

```diff
@@ -404,12 +408,10 @@ public static boolean conditions(final Profile profile, final ConditionID... con
      */
     @SuppressWarnings("PMD.NPathComplexity")
     public static boolean condition(final Profile profile, final ConditionID conditionID) {
-        // null check
```

Comment:
This (and all the other comment removals) is cleanup and not part of adding the JRE to the debug log. Maybe it would have been better to do that in a separate PR.

Reply:
I agree that normally this should have been split but it's an unnecessary overhead now.